app/src/main/java/com/yacgroup/yacguide/database/DatabaseWrapper.kt

```diff
@@ -29,7 +29,7 @@ class DatabaseWrapper(context: Context) {
 
     companion object {
         const val UNKNOWN_NAME = "???"
-        const val INVALID_ID = R.integer.unknown_id
+        const val INVALID_ID = -1
```

Comment:
Same bug here: You'd need `resources.getInteger(R.integer.unknown_id)` to retrieve the value. However, the database should not hold information about the calling context, so I switched back to an id separate from the default region id.